clients/src/test/java/org/apache/kafka/common/requests/MetadataRequestTest.java

```diff
@@ -20,6 +20,7 @@
 import org.apache.kafka.common.errors.UnsupportedVersionException;
 import org.apache.kafka.common.message.MetadataRequestData;
 import org.apache.kafka.common.protocol.ApiKeys;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
nit: We usually put static imports after the imports. Could we move it down? I think that we could put it right before the other static imports.

Reply:
done